provider/resource_service_test.go

```diff
@@ -278,6 +278,48 @@ func TestAccResourceServiceKpisHandleUnknownTemplateId(t *testing.T) {
 	})
 }
 
+func TestAccResourceServiceHandleUnknownKpiBaseSearchId(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { testAccPreCheck(t) },
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testAccCheckResourceDestroy(resourceNameService, "test_kpis_2"),
```

Comment:
This function takes ITSI object titles (which should always start with `TestAcc_`), not the terraform resource instance names 😉

Reply:
thank you, makes sense :D